stdlib/array.gr

```diff
@@ -601,6 +660,9 @@ provide let contains = (search, array) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to search
  * @returns `Some(element)` containing the first value found or `None` otherwise
+ * 
+ * @example
+ * Array.find(a => a == 7, [> 5, 6, 7, 8, 9]) // Some(7)
```

Comment:
Same As Above:
```suggestion
 * Array.find(a => a == 7, [> 5, 6, 7, 8, 9]) == Some(7)
```

might also make sense to add a counter example here if it isnt found it returns `None`

Reply:
See previous comment about discussion with Oscar and simplicity of in line.  Again if one is going to use a Boolean comparison in an example you must provide the answer and not infer it is true...or ad the assert.....